include/fty_common_db_asset.h

```diff
@@ -48,6 +48,11 @@ namespace DBAssets {
     int64_t
     name_to_asset_id (std::string asset_name);
 
+// name_to_asset_id_check_type: converts asset internal name to database id if asset is of a specified type
+// returns value < 0 if error ocurres
+    int64_t
+    name_to_asset_id_check_type (std::string asset_name, uint16_t asset_type);
```

Comment:
Switch `std::string asset_name` to `const std::string& asset_name`.

Reply:
Fixed.

Reply:
Could you please explain to me how this function is supposed to work without specifying which DC such asset belongs to, when names could be conflicting between DCs?

Reply:
Many more functions will have to be rewritten, and do not want to pollute this PR with the changes.